config/zones/US-SW-EPE.yaml

```diff
@@ -172,74 +174,74 @@ fallbackZoneMixes:
       datetime: '2017-01-01'
       value:
         battery discharge: 0
-        biomass: 0.000003853100112863885
+        biomass: 3.853100112863885e-06
```

Comment:
Why were all these numbers updated? Please do it in a separate PR so we can keep track of things and point to a specific PR in future :)

Reply:
The value is the same but I think the Yamel package or prettier formatted them.